extensions/sql-migration/src/api/azure.ts

```diff
@@ -35,6 +35,14 @@ export async function getSubscriptions(account: azdata.Account): Promise<Subscri
 
 export type AzureProduct = azureResource.AzureGraphResource;
 
+export type ResourceGroup = azureResource.AzureResourceResourceGroup;
```

Comment:
Why are you redefining the type here anyways? That seems unnecessary - just use the azure resource type directly